tracer.go

```diff
@@ -116,7 +114,7 @@ type Tracer interface {
 
 // StartSpanOptions allows Tracer.StartSpan() callers and implementors a
 // mechanism to override the start timestamp, specify Span References, and make
-// Tags available at Span start time.
+// a single Tag or multiple Tags available at Span start time.
```

Comment:
I know what you're getting at, but this prose reads redundantly, and isn't quite correct. One can pass multiple singular `Tag` to this function, which makes it multiple `Tag`'s, but not `Tags`. Do we really want both types to exist?